### PR TITLE
Increase x-site configuration symmetry #235

### PR DESCRIPTION
**Pull request by galderz**
* Make x-site configuration as symmetric as possible.
* Only site name should differ between configuration of different sites.
* Update documentation to new representation.
* Fix put HTTP response code in x-site Minikube README.
* Added create-cache, put and get steps to x-site OpenShift example.

### CODE REVIEW

**Comment by galderz** [comment]
@oraNod I've gone ahead and made the necessary changes to the document, can you have a look and see what you think?

**Comment by galderz** [comment]
Oh, one thing is missing: changing the design doc. I'll do later today.

**Comment by galderz** [comment]
CI didn't work cos of #237, that's being fixed in a separate PR.

**Comment by galderz** [comment]
Issue fixed, restarting CI

**Comment by galderz** [comment]
@rigazilla Can you review this?

File: manual/xsite-minikube/Makefile [hunk 1]
```diff
@@ -77,18 +77,21 @@ undeploy-operator:
 define undeploy-operator
 minikube profile ${1}
 kubectl delete deployment infinispan-operator || true
+kubectl delete role infinispan-operator || true
+kubectl delete rolebinding infinispan-operator || true
+kubectl delete serviceaccount infinispan-operator || true
```
Comment (rigazilla) [inline manual/xsite-minikube/Makefile]: should we also run `kubectl delete crd infinispans.infinispan.org` ?

Comment (rigazilla) [reply manual/xsite-minikube/Makefile]: and also added to the openshift makefile?

Comment (galderz) [reply manual/xsite-minikube/Makefile]: Fixed

**Comment by galderz** [comment]
@rigazilla I've fixed your last comment and squashed commits.

**Comment by rigazilla** [comment]
thank you @galderz !

**Pull request merged**